frontend/src/components/Logout.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { actions } from '../store';
+
+/* Logout */
+class _Logout extends Component {
```

Comment:
same comment with other components. it's better to name the class/component the same as the filename.